projects/v2/src/app/modules/root/root.component.ts

```diff
@@ -379,6 +396,25 @@ export class RootComponent implements OnInit, OnDestroy {
     this.dialog.open(LoadingComponent, config);
   }
 
+  checkLinkFormat(url: string) {
```

Comment:
I wouldn't call this 'checking' the link format, probably better to call `parseSheetUrl` ... also need to have a return type on your function (not `any` lol).